### ISSUE TITLE
Incorporate CKEditor accessibility improvements

### ISSUE THREAD

**Issue by crism** [anvc/scalar#99]
There is [a modified version of CKEditor](https://github.com/a11yfirst/distribution) available, designed for accessibility best-practices. It should be a drop-in substitute for the version currently distributed with Scalar.

**Comment by crism** [comment]
@plusperturbatio, can you help me understand what you did to upgrade CKEditor from 4.4.7 to 4.7.0? If we can bring it up to 4.10.0, we should be able to cut over horizontally to the a11yfirst modifications, but I’m having trouble understanding the local customizations.

**Comment by plusperturbatio** [comment]
Basically, I went through and updated each plug-in we were using and
updated them where it didn't break the editor (some plugins like codemirror
seemed to be a little problematic...) And then I updated to the newest Dev
(this is important, since ckeditor ships with a minified version of its
codebase, which makes modifications super difficult.) I then went through
the files one by one to see if we had made any edits to them previously and
re-implemented those edits again within the new versions. Finally, there
were some structural changes to the html/css used in some of the plugins,
which necessitated some additions to the bootstrap theme css (since that
theme is no longer being developed)

We originally tried updating ckeditor to the newest version, but ran into
some issues, specifically with the source code view on the edit page - so
we instead chose the newest version that fixed some issues in Safari while
not breaking some of the other plugins.

Updating to 4.10.* may be possible, but would need a lot of testing to make
sure that the various modifications we've made can still function and that
the plugins we're relying on still work.

**Comment by crism** [comment]
Thanks, @plusperturbatio. I will try to see if I can isolate the a11yfirst changes; it may be possible to backport those to 4.7.0.
